src/libaktualizr/primary/aktualizr.h

```diff
@@ -83,6 +83,14 @@ class Aktualizr {
    */
   DownloadResult Download(const std::vector<Uptane::Target>& updates);
 
+  /**
+   * Get target downloaded in Download call. Returned target is guaranteed to be verified and up-to-date
```

Comment:
I think these guarantees are not actually fulfilled in the current implementation (SQLStorage). We even commit changes in partial downloads.

Reply:
OK, it's not true yet, but in Serhiy's PR read handle is throwing on incomplete file (https://github.com/advancedtelematic/aktualizr/pull/972/files#diff-01a41e87d80cc7d2239a7088d4944b83R1087). I would prefer returning `nullptr`, but that is another story.

Reply:
Ok fair enough.

You can also bundle up a changelog update under an `Added` section if you're at it :).

Reply:
Another thing that is missing for that to be totally true is that we don't clean-up old target files.

Reply:
Yes. So you'd like to use this api now and plan enhancements to the target files handling as soon as possible?

Reply:
Yes, I will also change the docstring to mention that the guarantee is not really true yet, but we really want to be there =)

Reply:
I don't throw exception when we open partially downloaded image anymore. It is needed to update multipart hasher.